docs/DeveloperGuide.md

```diff
@@ -754,18 +967,147 @@ testers are expected to do more *exploratory* testing.
+       Expected: No applicant is deleted. Error details shown in the response area. Applicant list in working area remains the same.
+
+    1. Other incorrect delete commands to try: `delete`, `delete x`, `delete a` (where x is larger than the list size)<br>
+       Expected: Similar to previous.
+
+### Editing an applicant's status
+
+1. Editing an applicant's status while all applicants are being shown
+
+    1. Prerequisites: List all applicants using the `list` command. Multiple applicants in the list.
+
+    1. Test case: `status 1 s/o`<br>
+       Expected: The status of the first applicant is updated to *OFFERED*. Updated details of the applicant shown in the response area.
+       Working area shows the updated list of applicants.
+
+    1. Test case: `status 1 s/l`<br>
+       Expected: No applicant's status is edited. Error details shown in the response area. Applicant list in working area remains the same.
+
+    1. Other incorrect edit status commands to try: `status`, `status x s/o`, `status 1 s/` (where x is larger than the list size)<br>
+       Expected: Similar to previous.
+
+### Adding an interview to an applicant
+
+1. Adding an interview to an applicant while all applicants are being shown
+
+    1. Prerequisites: List all applicants using the `list` command. Multiple applicants in the list.
+
+    1. Test case: `addi 1 t/technical r/8.6`<br>
+       Expected: A technical interview with rating 8.6 is added to the first applicant in the list. Updated details of the applicant shown in the response area.
+       Working area shows the updated list of applicants.
+
+    1. Test case: `addi 0`<br>
+       Expected: No interview is added to any applicant. Error details shown in the response area. Applicant list in working area remains the same.
+
+    1. Other incorrect add interview commands to try: `addi`, `addi x`, `addi r/6.0` (where x is larger than the list size)<br>
+       Expected: Similar to previous.
+
+### Editing an interview of an applicant
+
+1. Editing an interview to an applicant while all applicants are being shown
+
+    1. Prerequisites: List all applicants using the `list` command. Multiple applicants in the list. The first applicant has at least one interview.
+
+    1. Test case: `editi 1 i/1 t/technical r/8.6`<br>
+       Expected: The first interview of the first applicant in the list is updated to a technical interview with rating 8.6. Updated details of the applicant shown in the response area.
+       Working area shows the updated list of applicants.
+
+    1. Test case: `editi 0`<br>
+       Expected: No interview is added to any applicant. Error details shown in the response area. Applicant list in working area remains the same.
+
+    1. Other incorrect edit interview commands to try: `editi`, `editi x`, `editi 1 i/x t/technical`, `editi 1 i/1 r/6.0` (where x is larger than the list size)<br>
```

Comment:
i think `editi 1 i/1 r/6.0`  is a valid command, maybe change it to `editi 1 i/0 r/6.0` or `editi 1 i/x r/6.0` ?